### ISSUE TITLE
Android Gradle 7.1.0 提示 com.android.tools.r8.internal.XI0: Method name '<get-conditionType>' in class

### ISSUE THREAD

**Issue by wangjieares** [Krosxx/Android-Auto-Api#53]
com.android.tools.r8.internal.XI0: Method name '<get-conditionType>' in class 'cn.vove7.andro_accessibility_api.api.View_finder_apiKt$sam$cn_vove7_auto_core_viewfinder_MatchCondition$0' cannot be represented in dex format.

**Comment by wangjieares** [comment]
更新最新 4.1.3 已经解决了
